attest/verifier/src/lib.rs

```diff
@@ -26,7 +26,7 @@ pub use crate::status::{Kind as StatusVerifier, MrEnclaveVerifier, MrSignerVerif
 cfg_if::cfg_if! {
     if #[cfg(feature = "sgx-sim")] {
         /// The build-time generated mock IAS signing root authority
-        pub const IAS_SIM_ROOT_ANCHOR: &str =include_str!("../data/sim/root_anchor.pem");
+        pub const SIM_ROOT_ANCHOR: &str =include_str!("../data/sim/root_anchor.pem");
```

Comment:
mega nit
```suggestion
        pub const SIM_ROOT_ANCHOR: &str = include_str!("../data/sim/root_anchor.pem");
```

Reply:
Thanks, I'm surprised cargo fmt didn't catch that.